src/events/guildMemberAdd.ts

```diff
@@ -18,9 +18,10 @@ export default TypedEvent({
             .setColor("ORANGE")
             .setTitle("New Member")
             .setDescription(
-                `Welcome ${member.user.username} to the conaticus server\n` +
-                    "Use `/rolemenu` to choose your pings and languages roles\n" +
-                    "Enjoy your stay!"
+                `Welcome ${member.user.username} to Conaticus' discord server\n` +
+                    "Use `/rolemenu` to edit your notification, language and colour roles.\n" + 
+                    "Be sure to read our #rules and feel free to introduce yourself in #introductions." +
```

Comment:
Consider swapping these to prioritise rules rather than roles.

Otherwise, this doesn't look too bad.

```suggestion
                    "Be sure to read our #rules and feel free to introduce yourself in #introductions." +
                    "Use `/rolemenu` to edit your notification, language and colour roles.\n" + 
```